javascripts/application.js

```diff
@@ -110,6 +110,14 @@ $(document).ready(function() {
                       $(this).text(
                           text == "Show Me >" ? "Hide Image" : "Show Me >");
   }));
+
+  // Add anchorJS anchors; see https://github.com/18F/content-guide/issues/149
+  anchors.options.placement = 'right';
+  anchors.add();
+  anchors.remove(".logo");
+  anchors.remove("#title-18f-content-guide");
+  anchors.remove("ol li h1");
```

Comment:
Hmm, instead of assigning an id of `title-18f-content-guide` and doing those last two separate `anchors.remove()` calls--the second of which could be confusing to folks who find this code later--what do you think of having just one `anchors.remove(".layout-table-of-contents h1")`?  It feels more semantic and readable to me because the use of `.layout-table-of-contents` makes it a little more clear that it's meant to un-anchor the entries on the TOC page, rather than any page in general.

If you *do* have to keep something on the `title-18f-content-guide` to make it selectable from JS, I wonder if using a boolean data attribute, say, `data-anchorjs-ignore`, might be a bit more maintainable--it also means that making *any* header ignored just requires adding that one data attribute (no extra JS required) which could be nice for the future too.